docs/quickstart.rst

```diff
@@ -2,17 +2,84 @@
 Quickstart
 ==========
 
+Requirements
+============
+
+This application makes use of the AbstractBaseUser model from django.
+Generally, you should be able to use this application with any Django project 
+that uses the default User model.
+If you are using a custom User model, you will need to make sure that it is 
+compatible with the AbstractBaseUser model.
+Otherwise you can customize things through the username and email properties on 
+your custom user model.
+
 Installation
 ============
 
-Install from PyPI with pip:
+1. Install from PyPI with pip:
+
+   .. code-block:: bash
+
+      pip install maykin-auth-matrix
+
+2. Add ``maykin-auth-matrix`` to ``INSTALLED_APPS`` in
+   your Django project's ``settings.py``:
+
+   .. code-block:: python
+
+      INSTALLED_APPS = (
+          "django.contrib.admin",
+          ...,
+          "auth_matrix",
+      )
+
+3. Run ``python manage.py migrate`` to create the necessary database tables.
+
+4. Add the URL to your Django project's ``urls.py``:
+
+   .. code-block:: python
 
-.. code-block:: bash
+      from django.contrib import admin
+      from django.urls import path, include
 
-    pip install auth_matrix
+      from auth_matrix.views import AuthMatrixView
 
+      urlpatterns = [
+          path(
+              "admin/auth_matrix/",
+              include(("auth_matrix.admin_urls", "auth_matrix"), namespace="auth_matrix"),
+          ),
+      ]
```

Comment:
setting `app_name = "auth_matrix"` in the URLs file allows you to simplify this to:

```suggestion
      urlpatterns = [
          path("admin/auth_matrix/", include("auth_matrix.admin_urls")),
      ]
```